docs/developer/channels.mdx

```diff
@@ -387,7 +391,7 @@ Channels can be removed only when:
 [`channelDelete`](docs/api-reference/channels/mutations/channel-delete.mdx) mutation takes [input](docs/api-reference/channels/inputs/channel-delete-input.mdx):
 
 - `id`: ID of the Channel that will be deleted
-- `targetChannel`: all existing orders will be moved into this channel
+- `channelId`: all existing orders will be moved into this channel
```

Comment:
its a bug in docs, actually its `channelId`. It'd better if API has `targetChannel` though